### PR TITLE
Validate before getById 

### PR DESCRIPTION
**Pull request by Idokah**
Add validation before `getById`, export `getByIdFilter` to utils.

### CODE REVIEW

File: packages/velo-external-db-core/lib/service/schema_aware_data.js [hunk 1]
```diff
@@ -13,6 +14,7 @@ class SchemaAwareDataService {
     }
 
     async getById(collectionName, itemId) {
+        await this.validateFilter(collectionName, getByIdFilter(itemId))
```
Comment (noam-almog) [inline packages/velo-external-db-core/lib/service/schema_aware_data.js:17]: add test for this and also validate cases of invalid id, empty string/null value should throw an exception

File: packages/velo-external-db-core/lib/converters/query_validator.spec.js [hunk 2]
```diff
@@ -51,6 +51,24 @@ describe('Query Validator', () => {
         })
     })
 
+    describe ('validateGetById', () => {
+        test('should throw InvalidQuery if itemId is not defined', () => {
+            expect ( () => env.queryValidator.validateGetById(ctx.fieldArrWithId)).toThrow(InvalidQuery)
+            expect ( () => env.queryValidator.validateGetById(ctx.fieldArrWithId, '')).toThrow(InvalidQuery)
```
Comment (noam-almog) [inline packages/velo-external-db-core/lib/converters/query_validator.spec.js:57]: also add a case '    '

File: packages/velo-external-db-core/lib/converters/query_validator.js [hunk 3]
```diff
@@ -15,6 +16,11 @@ class QueryValidator {
     
     }
     
+    validateGetById(fields, itemId) {
+        if (!itemId) throw new InvalidQuery('A value must be provided for itemId')
+        this.validateFilter(fields, getByIdFilterFor(itemId))
```
Comment (noam-almog) [inline packages/velo-external-db-core/lib/converters/query_validator.js:21]: this logic seems funny, we are validating a filter we synthetically create, i don't see the value in this. WDYT ?
